apps/helloworld/values.yaml

```
@@ -1,6 +1,7 @@
 ---
 cluster:
   issuer: ca-issuer
```

```suggestion
  issuer: selfsigned-issuer
```

This has nothing to do with your PR, but something I've noticed that needs to be changed :)